docs/create_page.md

```diff
@@ -19,7 +19,7 @@ The basic AMP HTML page includes the following mark-up:
       {
         "@context": "http://schema.org",
         "@type": "NewsArticle",
-        "headline": "Open-source framework for publishing content everywhere",
+        "headline": "Open-source framework for publishing content",
```

Comment:
I think this meant to mean that this is a solution not bound to any particular platform.